pkg/cmd/issue/create/create.go

```diff
@@ -314,3 +309,20 @@ func createRun(opts *CreateOptions) (err error) {
 
 	return
 }
+
+func generatePreviewURL(apiClient *api.Client, baseRepo ghrepo.Interface, tb shared.IssueMetadataState) (string, error) {
+	if len(tb.Projects) > 0 {
+		var err error
+		tb.Projects, err = api.ProjectNamesToPaths(apiClient, baseRepo, tb.Projects)
+		if err != nil {
+			return "", fmt.Errorf("could not add to project: %w", err)
+		}
+	}
```

Comment:
Could the projects portion be moved into `WithPrAndIssueQueryParams` to avoid having to mutate the `tb.Projects` property at all? I know it was like that before your change, but since you're moving it around already, it could be a nice improvement.